com.unity.perception/Documentation~/Tutorial/Phase3.md

```diff
@@ -301,7 +301,7 @@ Once you have entered all the information, the block of code should look like th
 </p>
 
 
-* **Action**: Continue to the next code block and run it to download all the meta-data files from the generated dataset. This includes JSON files and logs, but does not include images (which will be downloaded later). 
+* **Action**: Continue to the next code block and run it to download all the meta-data files from the generated dataset. This includes JSON files and logs but does not include images (which will be downloaded later). 
```

Comment:
Not sure if "meta-data" is actually the correct way, but "metadata" just feels right to me lol